pom.xml

```diff
@@ -281,13 +281,13 @@ under the License.
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
-                <version>2.17</version>
+                <version>3.4.0</version>
```

Comment:
Any issues with upgrading to the latest i.e. to 3.6.0 https://maven.apache.org/plugins/maven-checkstyle-plugin/dependency-info.html

```
<plugin>
  <groupId>org.apache.maven.plugins</groupId>
  <artifactId>maven-checkstyle-plugin</artifactId>
  <version>3.6.0</version>
```
?

Reply:
no, good catch, upgraded, thx!